zenoblend/main.cpp

```diff
@@ -162,6 +166,46 @@ PYBIND11_MODULE(pylib_zenoblend, m) {
         return meshPtr;
     });
 
+    py::bind_vector<std::vector<float>>(m, "FloatVec3");
```

Comment:
This can treat `std::vector` as a `PyObject` which behaves like Python internal object `list`, without  really converting it into a `list`. I guess this will get better performance when dealing with a large buffer. c.f. https://pybind11.readthedocs.io/en/stable/advanced/cast/stl.html#binding-stl-containers